src/common/com/intellij/plugins/haxe/ide/generation/CreateGetterSetterFix.java

```diff
@@ -46,7 +42,7 @@ boolean accept(String name, Set<String> names) {
     }, GETTERSETTER {
       @Override
       boolean accept(String name, Set<String> names) {
-        return !names.contains(HaxePresentableUtil.getterName(name)) &&
+        return !names.contains(HaxePresentableUtil.getterName(name)) ||
```

Comment:
Not sure about this fix.  Won't this add a second getter or setter if only one exists in the first place?  (e.g. if there's already a setter, and we accept, we allow a new getter, as we should, /and/ we allow a second setter?)